godaddy.com.cpanel_titanemail.json

```diff
@@ -0,0 +1,39 @@
+{
+  "providerId": "godaddy.com",
+  "providerName": "GoDaddy",
+  "serviceId": "cpanel_titanemail",
+  "serviceName": "cPanel TitanEmail",
+  "version": 1,
+  "logoUrl": "https://www.domainconnect.org/wp-content/uploads/2020/02/godaddy_logo_new.png",
+  "description": "Enables a domain to work with TitanEmail",
+  "variableDescription": "",
+  "syncPubKeyDomain": "domain-attach.api.godaddy.com",
+  "records": [
+    {
+      "type": "MX",
+      "host": "@",
+      "pointsTo": "mx1.titan.email",
+      "priority": 10,
+      "ttl": 3600
+    },
+    {
+      "type": "MX",
+      "host": "@",
+      "pointsTo": "mx2.titan.email",
+      "priority": 20,
+      "ttl": 3600
+    },
+    {
+      "type": "TXT",
+      "host": "@",
+      "data": "v=spf1 include:spf.titan.email ~all",
+      "ttl": 3600
+    },
```

Comment:
It is preferred that Domain Connect templates use the SPFM record, not a TXT record for SPF.  Otherwise conflicting SPF records in templates would overwrite each other.

Reply:
Hi Arnold, these are pretty much mirror copies of existing templates (https://github.com/Domain-Connect/Templates/blob/master/secureserver.net.cpanel_flockmail.json) which have been updated to reflect the change in branding from Flock to Titan email. The existing ones are using TXT for SPF and also have the DMARC record

Reply:
You are right; these use dmarc.  And I see one other template from plesk using it.  If critical, leave it in.

But nobody is using SPF.  Convert the new ones to SPFM.  Will behave better.